src/io/flutter/logging/FlutterLogEntry.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2018 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.logging;
+
+import com.intellij.openapi.util.text.StringUtil;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class FlutterLogEntry {
+
+  @NotNull
+  private final String timestamp;
```

Comment:
Perhaps use a long instead? I assume that would be something like ms since the epoch. Elsewhere, would could then convert that to a user string for rendering.

Reply:
Adding a TODO to reconsider.  We may make this go away altogether as discussed above.